app/views/categories/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="flex flex-col items-center my-6 mx-4">
+  <h1 class="text-3xl font-medium mb-6">Your categories</h1>
+  <% if @categories.empty? %>
+    <p class="text-gray-600">No categories found</p>
+    <img src="https://cdn-icons-png.flaticon.com/512/7486/7486831.png" alt="No categories found" class="w-32 h-32 my-10">
+  <% else %>
+  <div class="grid grid-cols-3 gap-6">
+      <% @categories.each do |category| %>
+      <div class="flex flex-col items-center p-6 rounded-lg bg-white shadow-lg">
+        <%= link_to category.name, category_path(category), class: "text-xl font-medium mb-3" %>
+        <%= image_tag(category.icon, alt: "Icon pic", class: "w-12 h-12") %>
+        $<%= category.expenses.sum(:amount) %> spent
+      </div>
+    <% end %>
+    </div>
+  <% end %>
+  <%= link_to "Add new category", new_category_path, class: "bg-blue-400 text-white py-2 px-4 rounded-lg mt-6 hover:bg-blue-800" %>
+</div>
```

Comment:
- [ ] Please kindly ensure your design matches the original design for the categories page.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/219107525-7076867f-12a7-4188-9b87-ba802cd93fcb.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/219107076-18d94388-1256-4c3a-82c7-98208c53601c.png)

Reply:
Thanks @AnselemOdims, i fixed it 😉 
![Screenshot 2023-02-15 at 13 32 17](https://user-images.githubusercontent.com/45739963/219120748-c0d0f4da-04c2-4b54-bebe-052f9aefb56d.png)